layersvt/device_simulation.cpp

```diff
@@ -2555,6 +2471,12 @@ void JsonLoader::GetValue(const Json::Value &parent, const char *name, VkPhysica
         return;
     }
     DebugPrintf("\t\tJsonLoader::GetValue(VkPhysicalDeviceSamplerFilterMinmaxPropertiesEXT)\n");
+    if (!PhysicalDeviceData::HasExtension(&pdd_, VK_EXT_SAMPLER_FILTER_MINMAX_EXTENSION_NAME)) {
```

Comment:
Could you explain this change, this message was moved which is something I thought should be done so great.

Should the value be loaded with GET_VALUE_WARN when not supported?

Reply:
It probably shouldn't.  I can have it return if the extension isn't supported.